src/machine-learning/machine-learning.ts

```diff
@@ -288,22 +304,26 @@ export interface TFLiteModel {
 }
 
 
-/**
- * A Firebase ML Model input object
- */
-export class ModelOptions {
-  public displayName?: string;
-  public tags?: string[];
 
-  public tfliteModel?: { gcsTfliteUri: string; };
-}
 
+function getMaskFromOptions(options: ModelOptions): string[] {
+  const mask: string[] = [];
+  if (options.displayName) {
+    mask.push('display_name');
+  }
+  if (options.tags) {
+    mask.push('tags');
+  }
+  if (options.tfliteModel) {
+    mask.push('tflite_model.gcs_tflite_uri');
```

Comment:
You have `tfliteModel` in the body but `tflite_model` in the mask here. Same for `gcs_tflite_uri`.

Reply:
Yes, this is intentional. Node.js field names are in camelCase, but mask must be snake_case to match the proto.